src/classes/BGE_BatchGiftEntry_UTIL_TEST.cls

```diff
@@ -15,4 +51,43 @@ private with sharing class BGE_BatchGiftEntry_UTIL_TEST {
         System.assertEquals('apple ...', BGE_BatchGiftEntry_UTIL.truncateList(fruits, 1));
     }
 
+    /*********************************************************************************************************
+    * @description Tests handleMissingPermissions() methods
+    */
+    @isTest
+    static void testHandleMissingPermissions() {
+        List<DescribeFieldResult> dfrs = new List<DescribeFieldResult>();
+        dfrs.add(UTIL_Describe.getFieldDescribe('Account','Name'));
+        dfrs.add(UTIL_Describe.getFieldDescribe('Account','CreatedById'));
+        String expectedErrorMsg = Label.bgeFLSError + ' [Account Name, Created By ID]';
+
+        try {
+            BGE_BatchGiftEntry_UTIL.handleMissingPermissions(dfrs);
+        } catch (AuraHandledException ex) {
+            System.assertEquals(expectedErrorMsg, ex.getMessage());
+        }
+    }
+
+    /*********************************************************************************************************
+    * @description Tests checkFieldPermissionsNeededForBatch() methods
+    */
+    @isTest
+    static void testCheckFieldPermissionsNeededForBatch() {
+        Boolean hasError = false;
+        try {
+            BGE_BatchGiftEntry_UTIL.checkFieldPermissionsNeededForBatch();
+        } catch (AuraHandledException ex) {
+            hasError = true;
+            String errorMsg = ex.getMessage();
+            System.assert(errorMsg.contains(Label.bgeFLSError), 'Only perms errors should be thrown.');
+        } catch (Exception ex) {
+            hasError = true;
+            System.assert(false, 'Unhandled exception thrown.');
+        }
+
+        if (!hasError) {
```

Comment:
Upon further consideration, I don't think we need this block ... the assert(true) isn't giving us anything since it will always clear, and it's not exactly verifying that we get that far.